modules/fuzztest/20241025.0/presubmit.yml

```diff
@@ -0,0 +1,20 @@
+matrix:
+  platform:
+  - ubuntu2004
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_unix_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+    - --action_env=CC=clang
+    - --action_env=CXX=clang++
+    - --host_cxxopt=-std=c++17
+    - --cxxopt=-std=c++17
+    - --cxxopt=-Wno-c++20-extensions
+    build_targets:
+    - '@fuzztest//fuzztest:fuzztest'
```

Comment:
We should add `init_fuzztest` to the build targets to be tested too. Since it's used by quite a few people and we didn't catch it earlier.

Reply:
Another top level target to test is `@fuzztest//fuzztest:fuzztest_gtest_main`.

This is the one I noticed is broken in "20240813.0" due to the incorrect `dev_dependency = True` setting on the `googletest` dependency.

Reply:
I added `@fuzztest//fuzztest:fuzztest_gtest_main`, which includes `init_fuzztest` as a dependency.